TribalWars/src/Infrastructure/Repository/BuildingsQueueRepository.cs

```diff
@@ -0,0 +1,38 @@
+﻿using ApplicationCore.Entities;
+using ApplicationCore.Interfaces;
+using Infrastructure.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Repository
+{
+    public class BuildingsQueueRepository : BaseRepository, IBuildingsQueueRepository
```

Comment:
Methods inside that Repository relate to Context.BuildingQueues, not to BuildingsQueue, so the name should be changed to BuildingQueuesRepository